tools/monorepo-utils/src/ci-jobs/index.ts

```diff
@@ -23,11 +23,24 @@ const program = new Command( 'ci-jobs' )
 		'Base ref to compare the current ref against for change detection. If not specified, all projects will be considered changed.',
 		''
 	)
+	.option(
+		'-e --event <event>',
+		'Github event for which to run the jobs. If not specified, all projects will with changes be considered.',
```

Comment:
The wording here is unclear. Do you mean `If not specified, all events will be considered.`? The current code already considers all packages with changes.

Reply:
I was struggling to get the right wording here. Your suggestion is much better. Updated with [5008e80](https://github.com/woocommerce/woocommerce/pull/46922/commits/5008e805cbcaf5a8db8d99fa71496459f5eb8b5a)